### PR TITLE
Add post pick function 

### LINKED ISSUE #32: Add post pick function

### PR DESCRIPTION
**Pull request by JuKahl**

### CODE REVIEW

File: components/matchBox/TeamNames.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import styles from "./MatchBox.module.css";
+import { useUser } from "@auth0/nextjs-auth0";
 
 export type Teams = {
```
Comment (mewiemers) [inline components/matchBox/TeamNames.tsx:4]: which type works here? VoidFunctionComponent is it a Handler?

File: pages/pick.tsx [hunk 2]
```diff
@@ -2,13 +2,11 @@ import Head from "next/head";
 import styles from "../styles/Home.module.css";
```
Comment (mewiemers) [inline pages/pick.tsx]: remove unused style ;-)

**Review by mewiemers — APPROVED** [review]
GJ 🏈

File: samples.http [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 ### POST n new order
-POST http://localhost:3000/api/users
+POST http://localhost:3000/api/picks
 Content-Type: application/json
 
 {
-    "name": "Julian",
-    "email": "Julian@gameofthron.es",
-    "password": "asdf1231241234"
+    "user": "123123" ,
```
Comment (fre-ben) [inline samples.http:10]: Why not 'Willi' ? :(

**Review by fre-ben — COMMENTED** [review]
LookedGTM 🎈

**Pull request merged**